src/harness/rwcRunner.ts

```diff
@@ -155,6 +156,11 @@ namespace RWC {
                     compilerResult = output.result;
                 });
 
+                if (!tsconfigFile) {

```

Comment:
this seems like a strange place to do this.. i do not think you want to do this everytime we run a test. does not seem like the right place..

- i would say do that locally, get the tsconfig.json and check them in for all tests. 
- make it an error that every test has a tsconfig.json, otherwise fail. 
- update scripts/ior to generate one of these all the time when you extract a test to ensure new tests have them too

Reply:
> make it an error that every test has a tsconfig.json, otherwise fail.

I don't think there's point in this - we're only adding `tsconfig`'s for projects without them for our own ease-of-inspection. We want to keep testing the original way the compiler options were passed, ideally.

> update scripts/ior to generate one of these all the time when you extract a test to ensure new tests have them too

On second inspection, you should just be able to move this into `endRecord` in `loggedIO` - however you'll need to duplicate `tsconfigFile = ts.forEach(ioLog.filesRead, f => isTsConfigFile(f) ? f : undefined);` there, ofc. You should also be able to do `ts.parseCommandLine(recordLog.arguments)` to get the compiler options (since the log is still open and the files still accessible).
(Though having duplicate functionality in the rwcRunner, at least locally, is useful for generating the configs for existing tests!).